gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
```

Comment:
Why it needs to be gradle 5 ?

Reply:
Because the minimum the required by the gradle GraalVM plugin is 5.0, sorry i will remove this change as 6.3 would be able to support it.

Reply:
OK. We are closing this PR so please move all the (updated) files to investigation repo.